esi_leap/common/utils.py

```diff
@@ -18,3 +20,8 @@
 
 def get_resource_lock_name(resource_type, resource_uuid):
     return resource_type + "-" + resource_uuid
+
+
+def get_sha256_str(base_str):
```

Comment:
This is only ever called from `esi_leap/objects/console_auth_token.py`. Can it go there instead of `utils.py`? Trying to avoid `utils.py` becoming a dumping ground.

Reply:
Yep! It was in `utils.py` in nova because it was called from multiple places, but I guess it's unlikely we'll need to generate another hashed token.